### PR TITLE
BETA: Register hafi.is-a.dev

### PR DESCRIPTION
**Pull request by HafiTheCat**
Added `hafi.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/hafi.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	"owner": {
+		"username": "HafiTheCat",
+		"email": "",
+		"twitter": "",
+		"discord": "hafi_the_cat"
+	},
+	"description": "Website about about me",
+	"repo": "https://github.com/HafiTheCat/hafi_the_cat.github.io",
+	"record": {
+		"CNAME": "hafi_the_cat.github.io"
```
Comment (CuteDog5695) [inline domains/hafi.json:11]: site does not exist (404 page)

File: domains/hafi.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"owner": {
+		"username": "HafiTheCat",
+		"email": "",
+		"twitter": "",
+		"discord": "hafi_the_cat"
```
Comment (CuteDog5695) [inline domains/hafi.json:6]: Twitter section doesnt have a username, if you dont want to list a Twitter account then remove that line

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/hafi.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+	"owner": {
+		"username": "HafiTheCat",
+		"email": "",
+		"discord": "hafi_the_cat"
+	},
+	"description": "Website about about me",
+	"repo": "https://github.com/HafiTheCat/hafi_the_cat.github.io",
+	"record": {
+		"CNAME": "hafithecat.github.io/hafi_the_cat.github.io"
```
Comment (CuteDog5695) [inline domains/hafi.json:10]: CNAME cant have anything after / (remove / and the stuff after it and then ill merge it)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
seems fine to me now

**Pull request merged**